c/example-1-hello-world/README.md

```diff
@@ -0,0 +1,49 @@
+# Hello, world!
+
+This is the simplest contract in the repository, it adds 2 and 2 and 
+returns the sum as an external message. Since parsing of external messages 
+is not yet supported, this contract is for toolchain testing purposes
+only.
+
+## Methods
+
+### Method `compute`
+#### Input values
+None
+#### Output value
+* ABI type: `uint64`
+* C type: `unsinged`
+* Value description: 4 (sum of 2 and 2)
+#### Notes
+To call this method, one needs to send a message to the contract.
```

Comment:
To call this method, send a message to the contract.
All messages are built by tvm_linker.
To build a message, invoke the following command: